### PR TITLE
Iteration 1 ln

### PR DESCRIPTION
**Pull request by lnchambers**
Merge my branch yo

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 db/csv/*.csv
+.DS_store
```
Comment (lnchambers) [inline .gitignore:2]: Update .gitignore to cut out any .DS_store and tmp files

File: app/controllers/little_shop_app.rb [hunk 2]
```diff
@@ -4,23 +4,27 @@ class LittleShopApp < Sinatra::Base
     erb :dashboard
   end
 
-  get '/merchants/index' do
+  get '/merchants' do
```
Comment (lnchambers) [inline app/controllers/little_shop_app.rb:7]: Remove /index for restful routes

File: app/views/merchants/index.erb [hunk 3]
```diff
@@ -1,7 +1,13 @@
-<h1>Looking for a specific merchant?</h1>
+<a class="button" href="/merchants/new">New Merchant</a>
```
Comment (lnchambers) [inline app/views/merchants/index.erb:1]: Make links to CRUD

File: app/controllers/little_shop_app.rb [hunk 4]
```diff
@@ -4,23 +4,27 @@ class LittleShopApp < Sinatra::Base
...
 
-  post '/merchants/index' do
+  post '/merchants' do
     merchant = Merchant.create(params[:merchant])
-    redirect :"merchants/index/#{merchant.id}"
+    redirect :"merchants/#{merchant.id}"
   end
 
-  get '/merchants/index/:id' do
+  post '/merchants/find' do
```
Comment (lnchambers) [inline app/controllers/little_shop_app.rb:21]: Will need to change

File: spec/features/user_can_create_a_single_merchant_spec.rb [hunk 5]
```diff
@@ -1,12 +1,12 @@
 describe "As a user" do
   describe "when I go to the merchant index page" do
     it "I can create a new merchant" do
-      visit '/merchants/index/new'
+      visit '/merchants/new'
```
Comment (lnchambers) [inline spec/features/user_can_create_a_single_merchant_spec.rb:4]: Make it RESTful

File: spec/features/user_sees_all_merchants_spec.rb [hunk 6]
```diff
@@ -6,9 +6,41 @@
       Merchant.create(name: "Opakawagalaga")
       Merchant.create(name: "Opakawagalaga")
       Merchant.create(name: "Opakawagalaga")
-      visit '/merchants/index'
+      visit "/merchants"
 
       expect(page).to have_content("Opakawagalaga")
     end
+
```
Comment (lnchambers) [inline spec/features/user_sees_all_merchants_spec.rb:13]: Tests for buttons

File: spec/spec_helper.rb [hunk 7]
```diff
@@ -23,4 +19,8 @@
   c.after(:each) do
     DatabaseCleaner.clean
   end
+
```
Comment (lnchambers) [inline spec/spec_helper.rb:22]: Minor refactor for Capybara and make Capybara save path

**Review by annaroyer — APPROVED** [review]
cool

**Pull request merged**